lambdas/data-migration2/src/executions.ts

```diff
@@ -67,6 +68,8 @@ export const migrateExecutions = async (
   knex: Knex
 ): Promise<MigrationSummary> => {
   const executionsTable = envUtils.getRequiredEnvVar('ExecutionsTable', env);
+  const bucket = process.env.system_bucket;
+  const stackName = process.env.stackName;
```

Comment:
I'm inclined to use `envUtils.getRequiredEnvVar` to fetch both of these values. They should always be set, since we're the ones setting them on the lambda deployment config, so treating them as required should be safe

Reply:
More fundamentally, because of the value of this error logging for users, I don't think it should be optional

Reply:
Agreed